pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "topicctl_parser"
+version = "0.0.1"
+description = "Parses topicctl output and sends notifications"
+authors = [
+  { name = "Ben McKerry", email = "ben.mckerry@sentry.io" },
```

Comment:
i would omit authors from here. though usually i believe we use `oss@sentry.io` and not individuals